Lisp-Dep/fix-sbcl.lisp

```diff
@@ -0,0 +1,52 @@
+(in-package :cl-user)
+
```

Comment:
what do you need this file for? we use closer-mop for mop functionality. please remove this file from the PR.